# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: parser/html/nsHtml5TreeOperation.cpp
# Commit: 1954ce9669b8
# Full Hash: 1954ce9669b86b61c418c8286fbe0754ad2419cd
# Author: Adam Vandolder <avandolder@mozilla.com>
# Date: 2024-05-15 20:24:18
# Regressor Bug: 1890428
# File Overlap Count: 1
# Description:
#   Bug 1890428 - Add serialization support for Declarative Shadow DOM. r=webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D206955
# ==============================================================================

diff -r 3e45e3d15eed -r 1954ce9669b8 parser/html/nsHtml5TreeOperation.cpp
--- a/parser/html/nsHtml5TreeOperation.cpp	Wed May 15 03:01:18 2024 +0000
+++ b/parser/html/nsHtml5TreeOperation.cpp	Wed May 15 03:01:19 2024 +0000
@@ -934,6 +934,7 @@
       nsIContent* root = nsContentUtils::AttachDeclarativeShadowRoot(
           *aOperation.mHost, aOperation.mShadowRootMode,
           aOperation.mShadowRootIsClonable,
+          aOperation.mShadowRootIsSerializable,
           aOperation.mShadowRootDelegatesFocus);
       if (root) {
         *aOperation.mFragHandle = root;